src/store/getters.ts

```diff
@@ -700,7 +700,7 @@ export const getEntityTypeOptions = (state: StateIF): Array<EntityI> => {
 
   let options: SelectOptionsI[]
   // special case for amalgamation - location may be CA or IN
-  if (isAmalgamation) options = [...bcOptions]
+  if (isAmalgamation(state)) options = [...bcOptions]
```

Comment:
This was preventing the `xproOptions` from showing since it was returning the function (which evaluates to true) instead of calling it.

Reply:
Yes, I also found this bug and fixed it 👍 

PS that's not the only big in this method.

Reply:
Outdated. Resolving.